### DISCUSSION TITLE
Daily 07/09

### DISCUSSION THREAD

**Discussion by NataliaAlvarezIspc** [grupo-ispc-2023-2-0/workflix-app#45] [Daily]
Con respecto al proyecto: 

- ¿Qué hice ayer?
- ¿Qué hice o haré hoy?
- Impedimentos o dificultades

**Comment by NataliaAlvarezIspc** [comment]
Ayer realicé el diagrama de clases inicial y lo subí a la wiki. También estuve leyendo el material que se encuentra en la plataforma del ISPC. Por la noche participé de la clase de inglés del instituto.
Hoy voy a continuar con lo pendiente de la plataforma y probar nuevamente con el ejercicio de app móviles que no pude terminar porque tenía un error en el código. Por la tarde tenemos clases de ciberseguridad y testing, en la que  voy a participar

**Comment by BraianTroncoso** [comment]
Hice un crud con android studio y probé algunas librerias.
